src/test/scala/com/workday/warp/arbiters/SmartNumberArbiterSpec.scala

```diff
@@ -245,6 +247,37 @@ class SmartNumberArbiterSpec extends WarpJUnitSpec with CorePersistenceAware {
     val smartNumber: Double = arbiter.smartNumber(responseTimes)
     smartNumber should equal (-1)
   }
+
+
+  /**
+    * Checks that our smart threshold tightly follows a decreasing trend, and loosely follows an increasing trend.
+    */
+  @Test
+  @Category(Array(classOf[UnitTest]))
+  def incorrectBehavior(): Unit = {
+
+    val arbiter: SmartNumberArbiter = new SmartNumberArbiter()
+
+    val responseTimes: List[Double] = this.readData("/rpca_sample_data.txt")
+    arbiter.smartNumber(responseTimes) should be(85.0 +- 1.0)
+    arbiter.smartNumber(responseTimes.reverse) should be(249.0 +- 1.0)
+
+    val responseTimes2: List[Double] = this.readData("/rpca_sample_data2.txt")
+    arbiter.smartNumber(responseTimes2) should be(505.0 +- 5.0)
+    arbiter.smartNumber(responseTimes2.reverse) should be(1233.0 +- 5.0)
+  }
+
+
+  /**
+    * Convenience method for reading a resource file as a time series.
+    *
+    * @param resourceName
+    * @return
+    */
+  private def readData(resourceName: String): List[Double] = {
```

Comment:
Since this is specific to the `incorrectBehavior` test, we could also stick it inside the test method instead. Totally optional.

Reply:
cool ill move it inside